src/openlcb/TractionThrottle.hxx

```diff
@@ -184,6 +184,10 @@ public:
     /// @return the controlled node (the train node) ID.
     /// @todo this function should not be here
     virtual openlcb::NodeID target_node() = 0;
+
+    /// Releases this throttles control over the train
+    /// @dispath is true to dispatch instead of release (WiThrottle specific)
+    virtual void release_train(bool dispath) = 0;
```

Comment:
The problem is that I cannot implement this correctly for OpenLCB.
Releasing a train is a fundamentally asynchronous operation, and this API is a synchronous API. There has to be time given for the stack to perform the network call, wait for response, handle timeouts and return a status.

All of this is included in the RELEASE_TRAIN api that's just above, line 106. So we'll have to start from that API style.

Reply:
OK, I'll switch to that approach.